pylab/website/templates/website/event_list.html

```diff
@@ -0,0 +1,27 @@
+{% load trans from i18n %}
+
+
+{% block content %}
```

Comment:
Since this template is used as part of other template by convention usually all templates of this kind are stored in `partials` directory. So please move it to `partials` so that anyone could now, that this supposed to be included into another template and not used as standalone template.

Also, since this is not a standalone template, you don't need `content` block.

As I understand your original intent was to make this a reusable template either as a standalone or as an partial. But let's follow [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it) principle and only move it to `partials` since now it is used as partial.